### PR TITLE
Fixing Some Broken Links

### PR DESCRIPTION
**Pull request by ElliotFriend**
In the `docs` dir, found and fixed some broken links using [markdown-link-check](https://github.com/tcort/markdown-link-check).

I ran the tool against the `api` directory, as well. Everything looks good there.

Signed-off-by: Elliot J. Voris <elliot@voris.me>

### CODE REVIEW

File: content/docs/start/list-of-operations.mdx [hunk 1]
```diff
@@ -602,7 +602,7 @@ Possible errors:
 | REVOKE_SPONSORSHIP_ONLY_TRANSFERABLE | -4 | Sponsorship cannot be removed from this ledgerEntry. This error will happen if the user tries to remove the sponsorship from a ClaimableBalanceEntry. |
 
 ## Set Trustline Flags
-[JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.setTrustLineFlags) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/SetTrustLineFlags.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#SetTrustLineFlags)
+[JavaScript](http://stellar.github.io/js-stellar-sdk/Operation.html#.setTrustLineFlags) | [Java](http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/SetTrustLineFlagsOperation.Builder.html) | [Go](https://godoc.org/github.com/stellar/go/txnbuild#SetTrustLineFlags)
```
Comment (Shaptic) [inline content/docs/start/list-of-operations.mdx:605]: Good catch, but I believe the casing is wrong here. This link 404s:

http://stellar.github.io/java-stellar-sdk/org/stellar/sdk/SetTrustLineFlagsOperation.Builder.html

while this link resolves (note TrustLine vs Trustline):

https://stellar.github.io/java-stellar-sdk/org/stellar/sdk/SetTrustlineFlagsOperation.Builder.html

File: content/docs/building-apps/connect-to-anchors/setup-for-anchored-assets.mdx [hunk 2]
```diff
@@ -349,7 +349,7 @@ Once we’ve found an Anchor that supports deposit and withdrawal, we can begin
 
 [1]: https://github.com/stellar/docs-wallet/tree/trustline
 [2]: https://www.stellarx.com/.well-known/stellar.toml
-[3]: https://www.anchorusd.com/.well-known/stellar.toml
+[3]: https://stablecoin.anchorusd.com/.well-known/stellar.toml
```
Comment (Shaptic) [inline content/docs/building-apps/connect-to-anchors/setup-for-anchored-assets.mdx:352]: Given the updated link, would you mind also updating the TOML file [above](https://github.com/stellar/new-docs/pull/418/files#diff-6d15707eed236ce95633b1176074c4110e744c816b8952117cbc7d93f6c863d2R305-R344) that it references?

**Review by Shaptic — APPROVED** [review]
Great catches, thank you!

**Review by Shaptic — CHANGES_REQUESTED** [review]
Sorry, wrong review state.

File: content/docs/building-apps/connect-to-anchors/setup-for-anchored-assets.mdx (hunk 2 above)
Comment (ElliotFriend) [reply content/docs/building-apps/connect-to-anchors/setup-for-anchored-assets.mdx:354]: I updated the StellarX one, too.

File: content/docs/start/list-of-operations.mdx (hunk 1 above)
Comment (ElliotFriend) [reply content/docs/start/list-of-operations.mdx]: Awwww, man! I totally didn't see that when looking in the Java docs. Thanks!

It appears most other SDKs and the Stellar docs in general use the TrustLine casing. Is that generally preferred? Is there a specific reason the Java SDK uses the lowercase L?

**Comment by ElliotFriend** [comment]
Thanks so much for the feedback! I've pushed another commit with those changes.

Incidentally, Atom likes to strip off extra whitespace at the end of lines when a file is saved. There were a couple instances of that in this commit. Is that kind of thing generally welcome in PRs, or is it discouraged? The whitespace is stripped in this commit, but I'm happy to undo that, if it's better.

Thanks again for helping me to poke around your codebase 👍🏻

**Comment by Shaptic** [comment]
@ElliotFriend I personally don't mind white-space trimming changes, since they're trivial to filter out of GitHub's diff viewer. As long as it's not like spaces -> tabs, a massive amount of change, or something else super significant, I'm fine with it.

**Review by Shaptic — APPROVED** [review]
Perfect! :+1: cc @rice2000

**Comment by rice2000** [comment]
Well done!

**Pull request merged**